vocabularies/PDF.xml

```diff
@@ -40,11 +40,17 @@
       </Term>
 
       <ComplexType Name="FeaturesType">
-        <Property Name="DocumentDescriptionReference" Type="Edm.String" Nullable="false">
+        <Property Name="DocumentDescriptionLocation" Type="PDF.DocumentDescriptionLocation" DefaultValue="0">
```

Comment:
Not sure here if value `0` is correct.  Would rather like to read it `entitySet`.

Reply:
If this points to an enum, don't we need a value help for it?

Reply:
I haven't seen value helps in the other documents either, so would guess this is not a use case here.

For concrete `DocumentDescription...` properties, they are not user-facing, but only filled programmatically.  One application, for example, will set `DocumentDescriptionLocation` to `header`, the other will set it to `entitySet`. But it's not the user that decides it.